css/style.css

```diff
@@ -372,4 +372,159 @@ main{
 	color: var(--color-white-100);
 	line-height: 2;
 }
+.main-block__inform {
+	padding: 120px 0;
+}
+.main-block__inform-title {
+	font-size: 16px;
+	color: var(--color-blue-200);
+	padding-left: 105px;
```

Comment:
Please, use priority in css properties, in all selectors 
* flex
* size
* color
* text